apps/testingaccessibility/src/styles/pricing.css

```diff
@@ -23,6 +23,27 @@
     @apply bg-moss-200/50 text-moss-800 hover:bg-moss-200/80;
   }
   [data-pricing-product-checkout-button]:disabled {
-    @apply bg-transparent my-8 shadow-inner bg-green-700 bg-noise text-white after:hidden;
+    @apply cursor-wait;
+  }
+
+  /* PPP box */
+  [data-pricing-product-ppp='0'] {
+    p {
+      @apply sr-only;
+    }
+    label {
+      @apply text-sm;
+    }
+  }
+  [data-pricing-product-ppp='1'] {
+    @apply max-w-sm p-5 border border-gray-200;
+  }
+  [data-pricing-product-ppp='2'] {
+    label {
+      @apply text-sm;
+    }
+    p {
+      @apply sr-only;
+    }
```

Comment:
I like this pattern for styling individual tier cards. `index` is all that is needed.